### PR TITLE
feat: Add welcome/leave messages

### LINKED ISSUE #278: ✨❓ Feature Request: Welcome message
**Description**

I find it necessary for Heptgram to have this feature
A welcome message

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Add welcome/leave messages (requires two channels named `welcome` and `leaves`).
  - Closes #278.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -7,9 +7,33 @@ const { token, colors, MongoDB, IDs, emoji } = require('./config.json');
...
+	  `${member} has just joined us! Say hello in the <#869607626185527366> channel! :grin:`
+	]
+
+	let msg = Math.floor(Math.random() * welcomemsgs.length)
+	getChannel(member.guild, "welcome")
+	  .send(welcomemsgs[msg].toString())
+})
+
+// Leaving messages. Requires a channel named `#leaves`.
+bot.on("guildMemberRemove", member => {
```
Comment (j-dogcoder) [inline main.js:27]: Hey! It's client.on instead of bot.on in this bot.

File: main.js [hunk 2]
```diff
@@ -7,9 +7,33 @@ const { token, colors, MongoDB, IDs, emoji } = require('./config.json');
 const antiAd = require('./features/anti-link');
 const antiInvite = require('./features/anti-invite');
 
-
 const client = new Discord.Client({ partials: ['MESSAGE', 'CHANNEL', 'REACTION'] });
 
+// Welcome messages. Requires a channel named `#welcome`.
+bot.on("guildMemberAdd", async member => {
```
Comment (j-dogcoder) [inline main.js:13]: Hey! It's client.on instead of bot.on in this bot.

**Review by j-dogcoder — CHANGES_REQUESTED** [review]
- See comments
- Can we make the channels and the messages configurable? In the issue, I think there is a guide about that using our mongo database.

**Comment by Panquesito7** [comment]
> the channels and the messages configurable

We can use a single `config.json` file for that if wanted.

>  In the issue, I think there is a guide about that using our mongo database.

Well, I don't know anything about MongoDB. 😅

**Comment by j-dogcoder** [comment]
I'd like it to be guild configurable if possible. I'll merge this, and open an issue for making it configurable.

**Review by j-dogcoder — APPROVED** [review]
#286

**Pull request merged**